README.md

```diff
@@ -295,6 +295,19 @@ Once the test node is running, start an infrastructure container in the same doc
 docker run -it --env-file env.list -v ${PWD}:/src -w /src --net aeternity aeternity/infrastructure
 ```
 
+Running/testing playbooks on localhost with docker-compose helpers.
```

Comment:
This addition is in the middle of other explanation. It should be at the end of the section as easier method than explained above it.